content/source/docs/extend/best-practices/depending-on-providers.html.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: "extend"
+page_title: "Extending Terraform: Depending on Providers"
+sidebar_current: "docs-extend-best-practices-depending-on-providers"
+description: |-
+  How to safely depend on providers and understand their interfaces.
+---
+
+## Depending on Providers
+
+Terraform's providers are a substantial amount of code, and occasionally it
+makes sense to depend on their functionality. The most straightforward and
+obvious way to depend on a provider is to depend on the Terraform CLI, but
+occasionally it makes sense to rely on a provider in a different context.
+
+This guide lays out the supported ways to interface with and depend on
+Terraform's providers. Unless the provider explicitly states otherwise, no
+other compatibility guarantees are provided.
+
+### Importing Providers as Go Modules
```

Comment:
```suggestion
## Do Not Import Providers as Go Modules
```

As written, this seems to imply that this is one of the options, but it's really an explanation for why you shouldn't.